src/Language/PureScript/CoreFn/Optimizer.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE FlexibleContexts #-}
+module Language.PureScript.CoreFn.Optimizer (optimize) where
+
+import Control.Monad.Reader (asks, MonadReader)
+import Language.PureScript.CoreFn.Ann (Ann)
+import Language.PureScript.CoreFn.Module (Module)
+import Language.PureScript.CoreFn.Optimizer.PassThroughCases
+import Language.PureScript.Options (Options, optionsNoOptimizations)
+
+-- |
+-- Apply a series of optimizer passes to CoreFn
```

Comment:
Could you please list the optimizations here, even if it's just the one for now?